invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/__tests__/DocumentPendingLoans.js

```diff
@@ -117,25 +127,34 @@ describe('DocumentPendingLoans tests', () => {
     const data = {
       hits: [
         {
-          loan_pid: 'loan1',
-          patron_pid: 'patron_1',
+          id: 1,
           updated: d,
           created: d,
```

Comment:
Sorry I saw somewhere above the `stringDate` wouldn't be better to name these `d` variables to the same name or they are not the same?